Year of the Voice Chapter 5 blog post (#30284)
home-assistant/home-assistant.github.io: source/_posts/2023-12-13-year-of-the-voice-chapter-5.markdown

For the final chapter of 2023, we have expanded the available types of voice commands to include weather, temperature, and to-do lists. Voice satellites are now aware of which area they're in, and more hardware/software options are available too.

Happy holidays!

<p class='img'>
<lite-youtube videoid="erf7HqTwCGs" videotitle="ESP32-S3-Box running Assist"></lite-youtube>
Assist running on the ESP32-S3-BOX-3.
</p>

<!--more-->

## The S3-BOX-3

Espressif recently released the [ESP32-S3-BOX-3], an update of the discontinued ESP32-S3-BOX (and "lite" variant). This "AIoT" development kit contains an ESP32-S3 chip, dual microphones, a small speaker, and a screen. Several docks are available in the box, which expose a USB-C power connector and GPIO pins for expanding the device.

<p class='img'>

> <lite-youtube videoid="erf7HqTwCGs" videotitle="Assist on the ESP32-S3-Box-3"></lite-youtube>
> Assist running on the ESP32-S3-BOX-3 with included artwork.

Reviewer: ```suggestion
<lite-youtube videoid="73QhFefsbbc" videotitle="Assist on the ESP32-S3-Box-3 with custom artwork"></lite-youtube>
Assist running on the ESP32-S3-BOX-3 with custom artwork.
```

We've extended the [Wyoming integration][wyoming] to communicate directly with [remote satellites][wyoming-satellite]. These satellites are automatically discovered, and can be configured much like ESPHome-based satellites with the ability to set an area and voice pipeline.

Several satellite modes are supported, including:

- **Always-on streaming** - satellite streams all audio to Home Assistant
- **Stream on speech** - only stream audio once speech is detected
- **Local wake word** - only stream audio when a wake word is detected locally

Audio clean up, such as automatic gain control and noise suppression, may be done in Home Assistant or on the satellite. A [Raspberry Pi Zero 2 W][rpi-zero-2w] has more than enough power to do local audio clean up and wake word detection, allowing you to have many satellites without straining your Home Assistant server. Reuse your old Raspberry Pi's, and start your journey with smart home voice control!

> 
> TODO: Add tutorial link

Reviewer: ```suggestion
```

---
layout: post
title: "Year of the Voice - Chapter 5"
description: "More hardware options and intents"

> date: 2023-12-07 00:00:00

Reviewer: ```suggestion
date: 2023-12-13 00:00:00
```

date_formatted: "December 13, 2023"
author: Michael Hansen
comments: true
categories: Assist
og_image: /images/blog/2023-12-13-year-of-the-voice-chapter-5/social.png
---

> TODO: Fix date
> 

Reviewer: ```suggestion
```

- Old school analog phones ([with an adapter][phone-tutorial])

Home Assistant users can now create multiple voice assistants by mixing and matching components of a voice "pipeline". Home Assistant Cloud subscribers automatically gain access to high-quality voice components in over 130 languages and dialects. Fully local components are available as well, such as our [Piper] text-to-speech system, allowing for 100% offline voice control.

In [Chapter 4], we added wake word processing directly into Home Assistant by leveraging the [openWakeWord] project. This allowed tiny voice satellites such as the [M5 ATOM Echo Development Kit][m5-tutorial] to offload wake word detection by streaming audio to a Home Assistant server. The community has been hard at work training a variety of [custom wake words](community-wake-words) that everyone can use to make their voice experience unique.

For the final chapter of 2023, we have expanded the available types of voice commands to include weather, temperature, and to-do lists. Voice satellites are now aware of which area they're in, and more hardware/software options are available too.

Happy holidays!

<p class='img'>

> <lite-youtube videoid="73QhFefsbbc" videotitle="ESP32-S3-Box with custom artwork"></lite-youtube>
> Assist running on the ESP32-S3-BOX-3 with custom artwork.

Reviewer: Let's put our artwork above the fold.
```suggestion
<lite-youtube videoid="erf7HqTwCGs" videotitle="ESP32-S3-Box running Assist"></lite-youtube>
Assist running on the ESP32-S3-BOX-3.
```

We've extended the [Wyoming integration][wyoming] to communicate directly with [remote satellites][wyoming-satellite]. These satellites are automatically discovered, and can be configured much like ESPHome-based satellites with the ability to set an area and voice pipeline.

Several satellite modes are supported, including:

- **Always-on streaming** - satellite streams all audio to Home Assistant
- **Stream on speech** - only stream audio once speech is detected
- **Local wake word** - only stream audio when a wake word is detected locally

Audio clean up, such as automatic gain control and noise suppression, may be done in Home Assistant or on the satellite. A [Raspberry Pi Zero 2 W][rpi-zero-2w] has more than enough power to do local audio clean up and wake word detection, allowing you to have many satellites without straining your Home Assistant server. Reuse your old Raspberry Pi's, and start your journey with smart home voice control!

> TODO: Add tutorial link

Reviewer: TODO
